Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -427,6 +427,14 @@ function Run_IMB_IO() {
 }
 
 function Main() {
+	LogMsg "Restarting waagent for ib0 device loading"
```

Comment:
why restart twice

Reply:
This is glitch in waagent. Originally it requires one time service restart, but it also needs recently another restart after VM reboot. After waagent 2.2.45, I will take a look at this part again.